platform/usng4j/platform-usng4j-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
+           http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <bean id="coordinateSystemTranslatorNad83Datum"
+          class="org.codice.usng4j.impl.CoordinateSystemTranslatorImpl">
+        <argument value="true"/>
+    </bean>
+
+    <bean id="coordinateSystemTranslatorNad27Datum"
+          class="org.codice.usng4j.impl.CoordinateSystemTranslatorImpl">
+        <argument value="false"/>
+    </bean>
+
+    <service id="CoordinateSystemTranslatorNad83Datum"
```

Comment:
Should we add `ranking`s to these services so that one deterministically is the default?